### ISSUE TITLE
ModuleNotFoundError: No module named 'drafttopic.utilities.fetch_wikiprojects'

### ISSUE THREAD

**Issue by BigBinnie** [wikimedia/drafttopic#64]
I have met the problem when running the `./utility fetch_wikiprojects --output <output_file_name.json>` in the README. It seems the code doesn't match well with the scripts. Could you please let me know how I can run the same script?

**Comment by halfak** [comment]
Oh!  Actually it looks like the README is out of date.  We have improved the way that data about wikiprojects gets pulled.  The README should be fixed, but in the meantime, if you tell me what you're trying to do, I can help out.

**Comment by halfak** [comment]
I've updated the readme in this change: https://github.com/wikimedia/drafttopic/pull/65

**Comment by BigBinnie** [comment]
> Oh! Actually it looks like the README is out of date. We have improved the way that data about wikiprojects gets pulled. The README should be fixed, but in the meantime, if you tell me what you're trying to do, I can help out.

Yeah! Thank you very much! I'm trying to get the category of a list of Wikipedia pages.

**Comment by halfak** [comment]
If you want a prediction, getting it from ORES is a good idea.  If you want to get the labeled data that we trained ORES models with, run `make datasets/enwiki.labeled_article_items.json.bz2` and then check out that dataset.

**Comment by halfak** [comment]
e.g. if I wanted a prediction for the article https://en.wikipedia.org/wiki/Albert_Einstein, I'd go to the history page and click on the most recent version to get https://en.wikipedia.org/w/index.php?title=Albert_Einstein&oldid=1120359753.  The number in the URL is a "revision ID".  I can provide that to ORES to ask for a prediction like this: https://ores.wikimedia.org/v3/scores/enwiki/1120359753/articletopic

If I look for all of the categories with a high probability, I see: 

STEM.Physics(99%)
STEM.STEM*(99%)
Culture.Biography.Biography*(87%)
STEM.Space(67%)
Geography.Regions.Europe.Europe*(66%)
Geography.Regions.Europe.Western Europe(47%)
Culture.Philosophy and religion(43%)

**Comment by BigBinnie** [comment]
Yeah! It's very helpful! Btw, do you know any API which could get the newest revision ID directly?

**Comment by BigBinnie** [comment]
> If you want a prediction, getting it from ORES is a good idea. If you want to get the labeled data that we trained ORES models with, run `make datasets/enwiki.labeled_article_items.json.bz2` and then check out that dataset.

I have run the make datasets/enwiki.labeled_article_items.json.bz2, but got:
`make: 'datasets/enwiki.labeled_article_items.json.bz2' is up to date.`
